### ISSUE TITLE
PDF Support ?

### ISSUE THREAD

**Issue by thereisnobandage** [VioletGiraffe/FasterThanSight#57]

**Comment by VioletGiraffe** [comment]
Sure, I'd be happy to add it if there is a reasonably simple way. Do you have a sample PDF file I should use for testing? Obviously, only plain text can be displayed.

**Comment by thereisnobandage** [comment]
I think it might be a bit of a problem when I think about it again, it seems difficult to clarify whether pdf books are legal or not.

**Comment by VioletGiraffe** [comment]
I see! That's not a concern for the software, but I'll have to use my own samples then.
Curiously, it looks like Qt can render PDF files in their entirety, but gives no way of parsing just the plain text.

**Comment by thereisnobandage** [comment]
good luck to you thanks for response 😊

**Comment by bridgebrain** [comment]
Sorry to hop in as some random, but maybe look into how accommodation for the blind software reads pdfs? I'll bet there's a module somewhere that does most of the file handling and text parsing out of the box

**Comment by VioletGiraffe** [comment]
That's an interesting idea, appreciate your input!

**Comment by thereisnobandage** [comment]
Not sure if it'll help, but I found this. [https://github.com/sybrexsys/VersyPDF](url)

**Comment by samridhsri** [comment]
does it support pdf now?

**Comment by VioletGiraffe** [comment]
@samridhsri : No, sorry, I did not have the time to do it. You are very welcome to contribute PDF support!
